scvi/module/base/_base_module.py

```diff
@@ -343,3 +366,133 @@ def create_predictive(
     def forward(self, *args, **kwargs):
         """Passthrough to Pyro model."""
         return self.model(*args, **kwargs)
+
+
+class JaxBaseModuleClass(linen.Module):
+    """Abstract class for Jax-based scvi-tools modules."""
+
+    def __init__(*args, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
why doesnt *args get passed here?

Reply:
because this is some randomly generated vscode fill in